rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1.java

```diff
@@ -32,9 +30,12 @@
 
 	private static final Logger API_LOG = LoggerFactory.getLogger(Constants.API_LOG);
 
-	@Autowired
 	private CpcFileService cpcFileService;
 
+	public CpcFileControllerV1(final CpcFileService cpcFileService) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public constructor by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)